__tests__/new.ts

```diff
@@ -151,6 +153,46 @@ export class CoolerArticleResource extends ArticleResource {
   }
 }
 
+export class TypedArticleResource extends CoolerArticleResource {
+  get tagString() {
```

Comment:
why make this a getter? that adds a lot of indirection and a performance hit, and only saves you typing `()` on the consumer side.

Reply:
It's a test - eventually I want to filter out getters in my type definitions. I don't think there's a way to currently do it.

Reply:
What for? a getter, an own property, and a method all should be showing up in type definitions.

Reply:
when i send a payload to update a Resource, I do not expect it to include methods.

```js
//good
update({ id }, {title: 'hi'});

// bad
update({id}, {title: 'hi', tagString: '123'});
```

Reply:
Ok - why not make tagString a normal method then? getters are generally to be avoided.

Reply:
I'm not sure how I could ban users from using getters.

Reply:
lol no, of course you can’t, but why would the use them in the first place?

Or by “experiment” do you mean you’re trying to make it ergonomic for them to do so?

Reply:
We try not to have opinions about things that aren't relevant to the principals of the library. If their types are wrong because they choose to use a getter, this would be a fault of the library.